index.html

```diff
@@ -51,6 +54,147 @@ <h1 class="light-green">I’m a software developer</h1>
+              <button type="button" class="see-demo-btn">See Project</button>
+            </div>
+          </div>
+          <div class="project-placeholder">
+            <div class="properties">
+              <h3 class="project-title .dark-blue">
+                Multi-Post Stories Gain+Glory
+              </h3>
+              <ul class="flex-div tags">
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">html</a></li>
+              </ul>
+              <button type="button" class="see-demo-btn">See Project</button>
+            </div>
+          </div>
+          <div class="project-placeholder">
+            <div class="properties">
+              <h3 class="project-title .dark-blue">
+                Multi-Post Stories Gain+Glory
+              </h3>
+              <ul class="flex-div tags">
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">html</a></li>
+              </ul>
+              <button type="button" class="see-demo-btn">See Project</button>
+            </div>
+          </div>
+          <div class="project-placeholder">
+            <div class="properties">
+              <h3 class="project-title .dark-blue">
+                Multi-Post Stories Gain+Glory
+              </h3>
+              <ul class="flex-div tags">
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">html</a></li>
+              </ul>
+              <button type="button" class="see-demo-btn">See Project</button>
+            </div>
+          </div>
+          <div class="project-placeholder">
+            <div class="properties">
+              <h3 class="project-title .dark-blue">
+                Multi-Post Stories Gain+Glory
+              </h3>
+              <ul class="flex-div tags">
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">html</a></li>
+              </ul>
+              <button type="button" class="see-demo-btn">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section id="about-me">
+        <img class="aboutme-img1" src="img/aboutme-img1.png" alt="png1" />
+        <h2 class="dark-blue">About me</h2>
+        <p class="dark-blue">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <img class="aboutme-img2" src="img/aboutme-img2.png" alt="png2" />
```

Comment:
- [ ] <b>REQUIRED:</b> Same thing goes here. The project requires to use background images, not img tags. Kindly fix it.
![scrnli_3_19_2022_8-03-35 PM](https://user-images.githubusercontent.com/45217364/159133067-1358bec8-160d-4c3f-9a45-aa9d83bed1da.png)

Reply:
Done